cws/dumpsv1/activity_dump.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.34.0
```

Comment:
Is it really necessary to change this ?

Reply:
the rakefile specifies latest, so every time you run `rake codegen` it may change these. I am in favor of hardcoding the version so version updates and functionality changes are not polluting each other's PRs